tests/expr_and_series/over_test.py

```diff
@@ -428,3 +428,17 @@ def test_over_without_partition_by(
     )
     expected = {"a": [1, 2, -1], "b": [1, 3, 4], "i": [0, 1, 2]}
     assert_equal_data(result, expected)
+
+
+def test_len_over_2369(constructor: Constructor, request: pytest.FixtureRequest) -> None:
```

Comment:
Why `2369`? :) I couldn't find any other `test_len_over` in this file 👀

```suggestion
def test_len_over(constructor: Constructor, request: pytest.FixtureRequest) -> None:
```

Reply:
😄 just the github issue where the bug was reported